shipment_advice/models/stock_move.py

```diff
@@ -12,6 +12,7 @@ class StockMove(models.Model):
         ondelete="set null",
         string="Planned shipment",
         index=True,
+        copy=False,
```

Comment:
This needs to reverted when https://github.com/OCA/stock-logistics-transport/pull/123 is merged.